lib/repo-listing.js

```diff
@@ -157,11 +157,36 @@ function markVisibilityByStatus(repos, statuses) {
 	});
 }
 
+/**
+ * Mark repositories as visible or not using repo brand support.
+ * @param {Array} repos - A list of repositories from the Repo Data API.
+ * @param {Boolean} [brand] - If internal is ticked, include it in the output.
+ * @return {Array} Returns the list of repositories.
+ */
+function markVisibilityByBrand(repos, brand) {
```

Comment:
I'm not sure about hiding components. Some components like `o-utils` are useful regardless of brand, and other non-branded components should be surfaced so people know they exist and can be branded. I've an idea that would be good to chat through Monday.